src/theory/evaluator.cpp

```diff
@@ -568,6 +665,22 @@ EvalResult Evaluator::evalInternal(TNode n,
           results[currNode] = EvalResult(res);
           break;
         }
+        case kind::BITVECTOR_UDIV:
```

Comment:
Hmm, technically, this is not correct if the second operand is zero. Maybe we should return `EvalResult()` in that case?